Tema7/src/main/java/TicketType.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Random;
+
+/**
+ * TicketType ticketType = // this should be enum value randomly generated.
+ */
+
+public enum TicketType {
+
+    FULL,
+    FULL_VIP,
+    FREE_PASS,
+    ONE_DAY,
+    ONE_DAY_VIP;
+
+
+    public static TicketType generateRandomTicket() {
+
+        return TicketType.values()[new Random().nextInt(values().length)];
+
+    }
+}
```

Comment:
still some work to be done on this part, please try extending the implementation as expected from the requests